core/src/main/java/org/mule/runtime/core/internal/event/AbstractEventContext.java

```diff
@@ -150,14 +150,14 @@ public final void success() {
   @Override
   public final void success(CoreEvent event) {
     if (isResponseDone()) {
-      if (LOGGER.isDebugEnabled()) {
-        LOGGER.debug(this + " response was already completed, ignoring.");
+      if (debugLogEnabled) {
+        LOGGER.debug("{} response was already completed, ignoring.", this);
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Invoke method(s) only conditionally. ([squid:S2629](https://sonarqube.kbuild.msap.io/coding_rules#rule_key=squid%3AS2629))

[See it in SonarQube](https://sonarqube.kbuild.msap.io/project/issues?id=mule-runtime-mule&pullRequest=7701&issues=AWkLjRWWotIXnmsUmgkz&open=AWkLjRWWotIXnmsUmgkz)

[ISSUE_KEY]: AWkLjRWWotIXnmsUmgkz